ereuse_devicehub/inventory/forms.py

```diff
@@ -234,15 +236,18 @@ def validate(self, extra_validators=None):
     def save(self, commit=True):
         if any([x == 'Error' for x in self.result.values()]):
             return
-        # result = []
         self.sync = Sync()
         schema = SnapshotSchema()
-        # self.tmp_snapshots = app.config['TMP_SNAPSHOTS']
-        # TODO @cayop get correct var config
-        self.tmp_snapshots = '/tmp/'
+        schema_lite = Snapshot_lite()
+        self.tmp_snapshots = app.config['TMP_SNAPSHOTS']
         for filename, snapshot_json in self.snapshots:
             path_snapshot = save_json(snapshot_json, self.tmp_snapshots, g.user.email)
             snapshot_json.pop('debug', None)
+            if snapshot_json.get('version') in ["2022.03"]:
```

Comment:
Please avoid hardcode constant "2022.03". 

Maybe could we create a method to perform this check?
`def is_wb_lite_snapshot(version:str) -> bool`

Reply:
NOTE: I see that current style uses `ereuse_devicehub/resources/enums.py`

Reply:
I put ```is_wb_lite_snapshot``` and I add in the config a new var:
```
WORKBENCH_LITE = ["2022.03"]
```